code/modules/projectiles/guns/ego_gun/aleph.dm

```diff
@@ -144,14 +144,16 @@
 			Can guns really bring peace and love?"
 	icon_state = "pink"
 	inhand_icon_state = "pink"
-	special = "This weapon has a scope, and fires projectiles with zero travel time. Damage dealt is increased when hitting targets further away."
+	special = "This weapon has a scope, and fires projectiles with zero travel time. Damage dealt is increased when hitting targets further away. Middle mouse button click/alt click to zoom in that direction."
```

Comment:
Going to test this on local.

Reply:
Tested it and it does work. I guess i could of just asked if you tested it but i just wanted to know if it did anything.

Reply:
yea i tested it and it seemed to be working correctly